www/Rakefile

```diff
@@ -24,206 +7,64 @@ require "fastly"
-
-      Log.info "Commit to gh-pages"
-      sh("git commit -m 'website update'")
-
-      require "inquirer"
-      if Ask.confirm("Ready to go, I have #{file_count} files at #{file_size}. "\
-                     "Do you want to push this live?", default: false)
-        Log.info "push to origin, this may take a moment"
-        sh("git push -u origin --force-with-lease gh-pages")
-      else
-        puts "Aborted."
+        if ENV["PUSH"]
+          sh "git commit -m 'website update'"
+          sh "git push -u origin gh-pages"
+        end
       end
-    end
 
-    sh("git checkout #{current_branch}")
-  end
-
-  desc "Release the site to the Netlify acceptance environment"
-  task :acceptance do
-    raise "NETLIFYKEY environment variable not set" unless ENV.key?("NETLIFYKEY")
-
-    Log.info "Cleaning the existing build directory"
-    Rake::Task["www:clean"].invoke
-
-    Log.info "Building the docs"
-    Rake::Task["docs"].invoke
-
-    Log.info "Building the site"
-    Rake::Task["www:build"].invoke
-
-    Log.info "Creating zip file of website contents"
-    Dir.chdir(File.join(Dir.pwd, "build")) do
-      sh("zip -r inspec-acceptance.zip *")
-    end
-
-    Log.info "Uploading to Netlify"
-    uri = URI.parse("https://api.netlify.com/api/v1/sites/inspec-acceptance.netlify.com/deploys")
-    request = Net::HTTP::Post.new(uri)
-    request["Content-Type"] = "application/zip"
-    request["Authorization"] = "Bearer #{ENV["NETLIFYKEY"]}"
-    request.body = File.read(File.join(Dir.pwd, "build", "inspec-acceptance.zip"), mode: "rb")
-
-    response = Net::HTTP.start(uri.hostname, uri.port, use_ssl: true) do |http|
-      http.request(request)
-    end
-
-    raise "Failed to upload to Netlify: #{response.code} -- #{response.body}" unless response.code == "200"
-
-    Log.info "Removing zip file"
-    File.unlink(File.join(Dir.pwd, "build", "inspec-acceptance.zip"))
-  end
-
-  desc "Flush the inspec.io cache at Fastly"
-  task :flush do
-    api_key    = ENV["FASTLY_API_KEY"]
-    service_id = ENV["FASTLY_SERVICE_ID"]
-
-    if api_key.nil? || service_id.nil?
-      puts ""
-      puts "WARNING: could not flush the Fastly cache for inspec.io."
-      puts "Please set FASTLY_API_KEY and FASTLY_SERVICE_ID environment variables"
-      puts "and run `rake www:flush`."
-      puts ""
-      next
-    end
-
-    Log.info "Flushing the Fastly cache"
-
-    begin
-      fastly_client  = Fastly.new(api_key: api_key)
-      fastly_service = Fastly::Service.new({ id: service_id }, fastly_client)
-      purge_result = fastly_service.purge_all
-    rescue => e
-      puts "ERROR: Fastly cache could not be purged: #{e.class} - #{e.message}"
-      next
-    end
-
-    if purge_result["status"] == "ok"
-      puts "Success! The Fastly cache is purged."
-    else
-      puts "ERROR: Fastly cache could not be purged: #{purge_result}"
+      sh("git worktree remove -f #{path}/gh-pages")
     end
   end
 end
```

Comment:
We are indeed behind a Fastly cache; I have a Fastly token and this code being removed here works to clear the cache. It looks like the default TTL is one hour. 

I think we should keep the Fastly clearing code.

Reply:
I was watching it refresh immediately whenever I pushed. Not sure what the difference is.

Reply:
Looks like cache-control is setting the pages to expire after 10 minutes. But I bet if you hit a page that was updated recently github will know that too and sets the right headers so fastly updates the cache.

```
$ curl -svL http://inspec.io 2>&1 |grep cache
< X-Served-By: cache-pao17445-PAO
< cache-control: max-age=600
< x-proxy-cache: MISS
< x-served-by: cache-pao17421-PAO, cache-pao17430-PAO
< x-cache: MISS, HIT
< x-cache-hits: 0, 1
```

https://docs.fastly.com/en/guides/controlling-caching

That said, I'm +1 on ditching this code.

As this code will be moved into automation in buildkite, we'll end up with something like this: https://expeditor.chef.io/docs/reference/actions/#purge-packages-chef-io-fastly

Reply:
I've added it back but I'd rather ditch it and remove another dependency.

I don't think anything we do on the website is mission critical enough to warrant the complexity.